queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -64,15 +66,28 @@ def process_email(email_msg: dict, flask_app: Flask):  # pylint: disable=too-man
 
     with flask_app.app_context():
         logger.debug('Attempting to process email: %s', email_msg)
-        email = None
+
+        token = AccountService.get_bearer_token()
+
         if email_msg['email']['type'] == 'bn':
             email = bn_notification.process(email_msg)
-        elif email_msg['email']['type'] == 'incorp':
-            email = incorp_notification.process(email_msg)
+        elif email_msg['email']['type'] == 'incorporationApplication':
+            email = incorp_notification.process(email_msg['email'], token)
         else:
             raise EmailException(f'Unrecognizable type: {email_msg["email"]["type"]}')
-        # TODO: send email via email service
-        logger.debug('NI: Sending email: %s', email)
+
+        email['recipients'] = 'kial.jinnah@gov.bc.ca'
```

Comment:
Not sure whether this is for testing?

Reply:
ooops haha

Reply:
that was for testing the email notification yeah